manager/cve_handler.py

```diff
@@ -358,6 +358,11 @@ def handle_get(cls, **kwargs):
                      {'arg_name': 'sap_sids', 'convert_func': None}]
         args = cls._parse_arguments(kwargs, args_desc)
         list_arguments = cls._parse_list_arguments(kwargs)
+
+        # check if advisory parameter comes with show_advisories parameter
+        if 'advisory' in args and args['advisory'] and 'show_advisories' not in args and not args['show_advisories']:
```

Comment:
`'show_advisories' not in args and not args['show_advisories']`

is going to crash with KeyError

Reply:
oh, I see, will fix it in a moment